core/src/main/kotlin/com/bumble/appyx/core/children/ChildEntry.kt

```diff
@@ -20,45 +18,20 @@ sealed class ChildEntry<T> {
         "$key@${javaClass.simpleName}"
 
     /** All public APIs should return this type of child which is ready to work with. */
-    class Eager<T>(
+    class Initialized<T>(
         override val key: RoutingKey<T>,
         val node: Node,
     ) : ChildEntry<T>()
 
-    /** Child representation for Lazy mode. */
-    class Lazy<T>(
+    class Suspended<T>(
         override val key: RoutingKey<T>,
-        private val resolver: Resolver<T>,
-        val buildContext: BuildContext,
-    ) : ChildEntry<T>() {
-        internal fun initialize(): Eager<T> =
-            Eager(key, resolver.resolve(key.routing, buildContext).build())
-    }
-
-    /** When to create child nodes? */
-    enum class ChildMode {
-
-        /** When routing state was changed. */
-        EAGER,
-
-        /** When node rendering was requested. */
-        LAZY,
-
-    }
+        val savedState: SavedStateMap?,
+    ) : ChildEntry<T>()
 
-    companion object {
-        fun <T> create(
-            key: RoutingKey<T>,
-            resolver: Resolver<T>,
-            buildContext: BuildContext,
-            childMode: ChildMode,
-        ): ChildEntry<T> =
-            when (childMode) {
-                ChildMode.EAGER ->
-                    Eager(key, resolver.resolve(key.routing, buildContext).build())
-                ChildMode.LAZY ->
-                    Lazy(key, resolver, buildContext)
-            }
+    /** Keep off-screen nodes in the memory? */
```

Comment:
Does this comment make sense for the enum as a whole, or just the KEEP enum?

Reply:
It a question and an answer is a value of enum.